src/backend/BSConfig.pm.template

```diff
@@ -358,4 +358,19 @@ our $local_registry_signatures_extension = 1;
 # network name required for services using a local proxy/cache like gemianbox
 # our $obs_service_network = "obs_services";
 
+if (-e '/etc/sysconfig/proxy') {
+  open my $proxysettings, '<', '/etc/sysconfig/proxy' or die "Unable to open file:$!\n";
+  my %hash = map { split /=|\s+/; } <$proxysettings>;
+  close $proxysettings;
+  our $proxy = $proxysettings{HTTP_PROXY};
```

Comment:
shouldnt this be '$hash{'HTTP_PROXY'}`?

Reply:
It have improved the code. Should now be OK.